index.js

```diff
@@ -73,7 +73,9 @@ module.exports = function (param) {
 
 			content = stripJson(content);
 
-			file.contents = new Buffer(JSON.stringify(content, null, "\t"));
+			let space = param.space || null
```

Comment:
maybe the default should be `\t` but anyway its a nice feature to set it from outside so we can use it for the build